run_properties.yml

```diff
@@ -1,3 +1,28 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
Why is this schema part of the actual run properties? Shouldn't it be in a run_properties_schema file?

Reply:
Addressed in
[[TECH-293] Move stage definitions to separate schema file](https://github.com/Vandebron/mpyl/pull/333/commits/76022a8c6e672b3e30b552eac917a948195ef79f) and [[TECH-293] Add documentation and health check for stage schema file](https://github.com/Vandebron/mpyl/pull/333/commits/e2ea40720b45db4ec8d39f4f2f9ffd5947affa0c)